src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java

```diff
@@ -50,8 +50,10 @@ public void testToEntity() {
         assertEquals(expectedQuestion.getShowGiverNameTo(), actualQuestion.getShowGiverNameTo());
         assertEquals(expectedQuestion.getShowRecipientNameTo(), actualQuestion.getShowRecipientNameTo());
         assertEquals(expectedQuestion.getShowResponsesTo(), actualQuestion.getShowResponsesTo());
-        assertEquals(expectedQuestion.getCreatedAt(), actualQuestion.getCreatedAt());
-        assertEquals(expectedQuestion.getUpdatedAt(), actualQuestion.getUpdatedAt());
+        assertNotNull(expectedQuestion.getCreatedAt());
+        assertNotNull(actualQuestion.getCreatedAt());
+        assertNotNull(expectedQuestion.getUpdatedAt());
```

Comment:
Same here.

Reply:
I'm sorry, can you clarify? Should I remove these lines?

Reply:
Yup. It follows the comment (https://github.com/TEAMMATES/teammates/pull/9619#discussion_r272906719) above.

Reply:
I've deleted the indicated lines.

Reply:
But you should not delete `assertNotNull(actualQuestion.getCreatedAt());` right?